src/Lobby.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import "./App.css";
+import GameHUD from "./GameHUD";
+import { checkLogin } from "./Login";
+import { useParams } from "react-router-dom";
+import io from "socket.io-client";
+
+interface ILobby {}
+
+const Lobby = (props: ILobby) => {
+  const [loggedIn, setLoggedIn] = React.useState<boolean>(false);
+  const [users, setUsers] = React.useState<string[]>([]);
+  const [email, setEmail] = React.useState<string>("");
+  const [socket, setSocket] = React.useState(() => io("/"));
+  const [update, setUpdate] = React.useState<string>("");
+  const { id } = useParams();
+
+  React.useEffect(() => {
+    const asyncOps = async () => {
+      const loginStatus = await checkLogin();
+      if (loginStatus.logged_in) {
```

Comment:
Unless there's a compelling reason for snake_case, I think we're better off defaulting to camelCase